src/Compilers/VisualBasic/Test/CommandLine/ErrorLoggerTests.vb

```diff
@@ -115,6 +115,7 @@ End Class
         ""title"": ""Unused local variable"",
         ""category"": ""Compiler"",
         ""isEnabledByDefault"": ""True"",
+        ""isSuppressedInSource"": ""False"",
```

Comment:
Interesting. Shouldn't we align the names that Roslyn uses with the format? Won't it be confusing otherwise? Is the format set in stone?